.gitmodules

```diff
@@ -218,3 +214,11 @@
 	path = next/external/stronghold.rs/develop
 	url = https://github.com/iotaledger/stronghold.rs
 	branch = dev
+[submodule "shimmer/external/identity.rs/develop"]
```

Comment:
```suggestion
[submodule "shimmer/external/identity.rs/0.7-alpha"]
```

Reply:
Renamed the whole submodule https://github.com/iota-wiki/iota-wiki/pull/760/commits/0c1c0bb9b5f8e0c583e274d4f8759ee32663343a